src/test/scala/me/jeffmay/neo4j/client/ws/TestGlobal.scala

```diff
@@ -0,0 +1,25 @@
+package me.jeffmay.neo4j.client.ws
+
+import com.typesafe.config.{Config, ConfigFactory}
+import me.jeffmay.util.akka.{NonBlockingCountingThrottler, TestGlobalAkka}
+
+import scala.concurrent.duration._
+import scala.util.{Failure, Success, Try}
+
+object TestGlobal {
+
+  lazy val config: Config = Try(ConfigFactory.load()) match {
+    case Success(loaded) => loaded
+    case Failure(ex) =>
+      println("\u001B[31m" +
+        s"""
+           |Could not load required configuration for tests. Encountered exception:
+           |${ex.getMessage}
+           |
+           |Run docker/start.sh to get a printout of the required configs.
+           |""".stripMargin + "\u001B[0m")
+      sys.exit(1)
```

Comment:
This kills `sbt` as well. I figure it isn't terrible since it is just test code, and my workflow is to just copy and paste the output from `docker/start.sh` (which will just print stuff if already started) and then run sbt again.